taxi/forms.py

```diff
@@ -0,0 +1,45 @@
+from django import forms
+from django.contrib.auth.forms import UserCreationForm
+from django.contrib.auth import get_user_model
+from django.core.exceptions import ValidationError
+
+from .models import Driver, Car
+
+
+def validate_license_number(license_number):
+    if len(license_number) != 8:
+        raise ValidationError("License number must be 8 characters long.")
+    elif not license_number[:3].isalpha() or not license_number[:3].isupper():
+        raise ValidationError("First 3 characters must be uppercase letters.")
+    elif not license_number[3:].isdigit():
+        raise ValidationError("Last 5 characters must be digits.")
+    return license_number
+
+
+class CarForm(forms.ModelForm):
+    drivers = forms.ModelMultipleChoiceField(
+        queryset=get_user_model().objects.all(),
+        widget=forms.CheckboxSelectMultiple,
```

Comment:
if you look at your screens you will see that checkbox widget hasn't been added.. could you check it, please?

Reply:
<img width="1437" alt="Screenshot 2022-11-09 at 13 36 17" src="https://user-images.githubusercontent.com/111740601/200820373-c8183d4e-f66b-493f-9685-5cbb1c13892e.png">
fixed) forgot to add `form_class` to the views